Library/Homebrew/test/support/fixtures/cask/Casks/with-installable.rb

```diff
@@ -11,11 +11,11 @@
             quit:       'my.fancy.package.app',
             login_item: 'Fancy',
             delete:     [
-                          "#{TEST_TMPDIR}/absolute_path",
```

Comment:
I’m fine either way with the indentation, but if we’re making it a fixed number of spaces instead of visual, shouldn’t we also change the first line?

To explain what I mean, on this file we have:

```ruby
quit:       'my.fancy.package.app',
login_item: 'Fancy',
delete:     [
```

But given the indentation change, don’t you think it should now be:

```ruby
quit: 'my.fancy.package.app',
login_item: 'Fancy',
delete: [
```

Reply:
> But given the indentation change, don’t you think it should now be:

I'm fine with that, it just won't be autocorrected.

Reply:
Did a quick check, and it seems `sub(/:\s+/, ': ')` should be enough to fix these in one swoop. I can do that (and do `audit` and `style` checks) after this is live.